htdocs/expedition/class/expedition.class.php

```diff
@@ -238,7 +245,7 @@ function create($user)
 		$sql.= ", ".(!empty($this->model_pdf)?"'".$this->db->escape($this->model_pdf)."'":"null");
 		$sql.= ")";
 
-		dol_syslog(get_class($this)."::create", LOG_DEBUG);
+		dol_syslog(get_class($this)."::create sql=".$sql, LOG_DEBUG);
```

Comment:
Well I think you should as you are making a PR.

In fact, it was announced in the dev list (http://lists.nongnu.org/archive/html/dolibarr-dev/2014-06/msg00001.html) but documentation is obviously something that we must take care of.